doc/source/whatsnew/v1.2.0.rst

```diff
@@ -298,7 +298,7 @@ Indexing
 Missing
 ^^^^^^^
 
-- Bug in :meth:`SeriesGroupBy.transform` now correctly handles missing values for `dropna=False` (:issue:`35014`)
+- Bug in :meth:`SeriesGroupBy.transform` and :meth:`DataFrameGroupBy.transform` now correctly handle missing values (:issue:`35014` and :issue:`35612`)
```

Comment:
missing values in the grouper? with dropna?

Reply:
rewrote, mentioned grouper. Also separated this entry from #35014